package/com.unity.formats.usd/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "dependencies": {
+        "com.unity.timeline" : "1.0.0"
+    },
+    "description": "The USD package provides an API to manipulate Universal Scene Description (.usd) files. USD is a 3D Scene format that scales well to large scenes and contains assets of different types (geometry, images, etc).", 
+    "displayName": "USD", 
+    "keywords": [
+        "usd" 
+    ], 
+    "name": "com.unity.formats.usd",
+    "repoPackagePath": "package/com.unity.formats.usd", 
+    "repository": {
+        "type": "git", 
+        "url": "ssh://git@github.com:Unity-Technologies/usd-unity-sdk.git"
+    }, 
+    "unity": "2018.3",
+    "unityRelease": "4f1",
+    "version": "0.0.1-preview.1"
```

Comment:
No objections.
Note that the .NET DLLs are versioned as 19.01, which matches USD. Making these match the package version probably makes most sense, however it will cause version numbers to go down, which will likely throw some warnings for existing users.